### PR TITLE
Add copy_text to InlineKeyboardButton

### PR DESCRIPTION
**Pull request by m2k7m**

### CODE REVIEW

File: pyrogram/types/bots_and_keyboards/inline_keyboard_button.py [hunk 1]
```diff
@@ -105,6 +110,7 @@ def __init__(
         self.callback_game = callback_game
         self.requires_password = requires_password
         self.pay = pay
+        self.copy_text = str(copy_text)
```
Comment (KurimuzonAkuma) [inline pyrogram/types/bots_and_keyboards/inline_keyboard_button.py:113]: Remove str() 🤔🤔🤔

File: pyrogram/types/bots_and_keyboards/inline_keyboard_button.py [hunk 2]
```diff
@@ -227,3 +233,9 @@ async def write(self, client: "pyrogram.Client"):
 
         if self.pay is not None:
             return raw.types.KeyboardButtonBuy(text=self.text)
+        
+        if self.copy_text is not None:
+            return raw.types.KeyboardButtonCopy(
+                text=self.text,
+                copy_text=self.copy_text
+            )
```
Comment (KurimuzonAkuma) [inline pyrogram/types/bots_and_keyboards/inline_keyboard_button.py:241]: Why only in read? What about write?!?!?!

**Review by KurimuzonAkuma — CHANGES_REQUESTED** [review]

**Comment by m2k7m** [comment]
done sir

File: pyrogram/types/bots_and_keyboards/inline_keyboard_button.py [hunk 3]
```diff
@@ -76,6 +76,10 @@ class InlineKeyboardButton(Object):
         pay (``bool``, *optional*):
             Change text of pay button.
             Available in :meth:`~pyrogram.Client.send_invoice`.
+        
+        copy_text (``str``,*optional*):
```
Comment (KurimuzonAkuma) [inline pyrogram/types/bots_and_keyboards/inline_keyboard_button.py:80]: Add space between , and *

**Review by KurimuzonAkuma — CHANGES_REQUESTED** [review]

**Review by KurimuzonAkuma — APPROVED** [review]

**Pull request merged**